contrib/html_demo/JupyterCode/13_image_similarity_export.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li>I had to add also "../../.." to sys.path.extend() so that utils_cv can get imported</li><li>needs an import os</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Completed